dags/springer/dag_pull_ftp.py

```diff
@@ -10,7 +10,7 @@
 
 @dag(
     start_date=airflow.utils.dates.days_ago(0),
-    schedule_interval="@hourly",
+    schedule_interval="30 */3 * * *",
```

Comment:
why this change?

Reply:
because in the issue was mentioned:
Check and enable if is missing the schedule for its publisher, it should be the same as in PRODUCTION